sdk/containerregistry/container-registry/samples/v1/javascript/package.json

```diff
@@ -25,6 +25,6 @@
   "dependencies": {
     "@azure/container-registry": "next",
     "dotenv": "latest",
-    "@azure/identity": "^1.1.0"
+    "@azure/identity": "2.0.0-beta.3"
```

Comment:
I think this will be overwritten next time the publish command is run. I think this change should happen in dev-tool or specified in package.json's `sampleConfiguration` entry:
```
"//sampleConfiguration": {
    // Any dependency version specifications in this setting will be preferred
    // over the dependencies specified in your ordinary dependencies or
    // devDependencies.
    "dependencyOverrides": {
      // For example, this setting will cause the generated sample packages
      // to include a dependency on a beta version of identity instead of the
      // version that your package depends on (currently ^1.1.0)
      "@azure/identity": "2.0.0-beta.3"
    }
  }
```

cc @witemple-msft

Reply:
or maybe I am wrong. @witemple-msft does the versions in the samples' package.json come from the package's package.json?

Reply:
I'd thought the sample's dependencies would be computed from the main package.json, no?

Reply:
I think you're right, ignore my request then unless Will says otherwise.

Reply:
@deyaaeldeen I think the "dependencyOverride" setting in the main package's package.json is when we want to specify an exception for the samples - in this case if we want samples to depend on a version of identity other than the package's dependency, then we'll use that setting.